scripts/bias_map.py

```diff
@@ -0,0 +1,182 @@
+import matplotlib.pyplot as plt
+import click
+from os import chdir, listdir
+from numpy import array, log, gradient, nan_to_num, argmax, power, exp, isinf, sort, convolve, ones, concatenate, arange, inf, max
+from uproot import recreate
+from uproot import open as op
+from scipy.optimize import curve_fit
+
+map = {
+   '10.73.137.104': {'apa': 1, 'fbk': [0, 1, 2, 3, 4, 5, 6, 7],
+       'hpk': [8, 9, 10, 11, 12, 13, 14, 15],'fbk_value':1060,'hpk_value':1560, 'DacV': [25.6, 25.8]},
+   '10.73.137.105': {'apa': 1, 'fbk': [0, 1, 2, 3, 4, 5, 6, 7, 8, 10, 13, 15],
+       'hpk': [17, 19, 20, 22],'fbk_value':1090,'hpk_value':1480, 'DacV': [25.8, 25.7, 25.8]},
+   '10.73.137.107': {'apa': 1, 'fbk': [0, 2, 5, 7],
+       'hpk': [8, 10, 13, 15],'fbk_value':1090,'hpk_value':1575, 'DacV': [25.8, 25.7]},
+   '10.73.137.109': {'apa': 2, 'fbk': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15],
+       'hpk': [16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32,
+           33, 34, 35, 36, 37, 38, 39],'fbk_value':1090,'hpk_value':1585, 'DacV': [25.8, 25.8, 25.7, 25.7, 25.6]},
+   '10.73.137.111': {'apa': 3, 'fbk': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+       16, 17, 18, 19, 20, 21, 22, 23],
+       'hpk': [24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39],
+       'fbk_value':1085,'hpk_value':1590, 'DacV': [25.7, 25.7, 25.6, 25.8, 25.7]},
+   '10.73.137.112': {'apa': 4, 'fbk': [],
+       'hpk': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18,
+           19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 34, 37, 39],
+       'fbk_value':0,'hpk_value':1580, 'DacV': [25.8, 25.6, 25.6, 25.9, 25.6]},
+   '10.73.137.113': {'apa': 4, 'fbk': [0,2,5,7], 'hpk': [],'fbk_value':1040,'hpk_value':0, 'DacV': [25.6]},
+}
+
+def derivative(bias, current):
+    x = current
+    f = log(x[len(x)/2:])
+    fpp=gradient(gradient(f))
+    fpp = nan_to_num(fpp)    
+    nbdv=bias[len(bias)/2:][argmax(fpp)]
+    return nbdv
+    
+def fit_pulse(t, t0, T, A, P):
+    x = (array(t) - t0+T)/T
+    return P+A*power(x,3)*exp(3*(1-x))
+
+def fit_pol(x, a, b, c, d, e, f):
+    return a*power(x, 5) + b*power(x, 4) + c*power(x, 3) + d*power(x, 2) + e*x + f
+
+def IV_relative(bias, current):
+    
+    b = bias/100
+    c = current
+
+    # The fit does not perform well if the files have a small amount of data
+    if len(bias)>20:
+
+        # Computing the relative derivative
+        der_y = gradient(c)
+        rel_y = der_y/c
+        rel_y = nan_to_num(rel_y)
+
+        # Checking if the data doesn't have infinite values
+        if sort(isinf(rel_y))[len(rel_y)-1] == False:
+
```

Comment:
Similarly here for the indentation. Something like:
```python
if sort(isint(rel_y))[-1] == True:
    return 0
```

At the same time, Python allows for negative indexing to start from the back.